examples/blend/main.go

```diff
@@ -0,0 +1,193 @@
+// Copyright 2023 The Ebitengine Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"image"
+	_ "image/png"
+	"log"
+	"math"
+
+	"golang.org/x/image/colornames"
```

Comment:
You don't have to use golang.org/x for white and black. Use `image/color`'s white and black.

Reply:
Ok

Reply:
Fixed